### DISCUSSION TITLE
Is there way to change prefix to suffix

### DISCUSSION THREAD

**Discussion by khackskjs** [nuxt-modules/i18n#1445] [Q&A] [answered]
I wanna use strategy `prefix` policy with not prefix but suffix.
means I wanna locale is always included in url, but I want it locates at the end of url

e.g) pages/about.vue => `/about/en` (not `/en/about`)

Is there anybody who knows what should I do?

**Comment by rchl** [comment, marked as answer]
This module doesn't support such mode and I don't foresee it being supported in the future.

You can use use this module with no_prefix strategy and locale auto-detection disabled and then implement something on top but I can't help you with that.

**Reply by khackskjs** [reply]
ok. I solved the problem with changing UX, as you mentioned one URL and dropdown menu for changing locale.
Thank you
